packages/frontend/src/components/LightdashVisualization/VisualizationProvider.tsx

```diff
@@ -65,11 +75,22 @@ export const VisualizationProvider: FC<Props> = ({
         pivotDimensions,
         resultsData,
     );
+
+    const chartTypeConfig =
+        chartType === chartConfigs?.type ? chartConfigs.config : undefined;
+
+    const { bigNumber, bigNumberLabel, setBigNumberLabel } = useBigNumberConfig(
```

Comment:
I think `onChartConfigChange` needs to be called somewhere to update the state of the configuration and have it persisted when you hit save.